crawl-ref/source/mapdef.cc

```diff
@@ -5315,9 +5315,8 @@ bool item_list::parse_single_spec(item_spec& result, string s)
         const string owner = replace_all_of(strip_tag_prefix(s, "owner:"),
                                             "_", " ");
 
-        COMPILE_CHECK(SPTYP_LAST_SCHOOL < SHRT_MAX);
-        result.props[RANDBK_DISC1_KEY].get_short() = disc1;
```

Comment:
(This is the old position of the `COMPILE_CHECK` which I moved to `spl-util.h`.)